js/views/itemEditVw.js

```diff
@@ -160,23 +160,26 @@ module.exports = Backbone.View.extend({
       contentType: false,
       processData: false,
       data: formData,
-      success: function(data) {
+      success: function(data) { //TODO: Have JQuery parse the JSON directly in ajax call
+        var errorModal, hashArray, imageArray;
```

Comment:
Should I commit this change now or do you only mean this a suggestion for the future?